Example/MultiSliderDemo/ViewController.swift

```diff
@@ -28,6 +30,10 @@ class ViewController: UIViewController {
             self.multiSlider.isValueLabelRelative = true
         }
 
+        DispatchQueue.main.asyncAfter(deadline: .now() + 4) {
+            self.updateValueLabel(text: "\(self.multiSlider.value)")
```

Comment:
Was this just for testing, or should it stay?

Reply:
I’ll remove all but the MultiSlider.swift change.